### ISSUE TITLE
[FEATURE] - Missing keywords

### ISSUE THREAD

**Issue by TheWitheredStriker** [alefragnani/vscode-language-pascal#137]
As described in #135. The following keywords are not currently highlighted by the grammar:

- assembler
- cdecl
- cppdecl
- constref
- helper
- nostackframe
- oldfpccall
- pascal
- register
- stdcall

I am creating a PR as we speak and will link it to this issue per the contribution guidelines.

Also missing from the highlighters are several keywords offered by another Pascal variant known as Oxygene (which uses e.g. `namespace` instead of `unit`, and `method` to define functions), but that's a different can of worms, and mainly relevant if this repo were to be used as new Pascal syntax highlighter for GitHub (see, again, issue 135). So I'll leave that for later discussion.

**Comment by alefragnani** [comment]
Hi @TheWitheredStriker ,

I played a bit with Oxygene back in the days when RemObjects were providing it as part of Delphi .NET, and it was quite promising TBH, as I would like to embrace the .NET ecosystem. But since they split relationship with Embarcadero, and no free product were available to play, I simply moved away. 

So yes, the extension _could_ be used to Oxygene as well (it certainly could already be, if they had a free tier product for me to play. You are free to open a PR with missing keywords, for sure.

Thank you
